src/components/qas/questions/qaFillableTable/FillableTableRangeInput.jsx

```diff
@@ -41,14 +48,47 @@ class FillableTableRangeInput extends React.PureComponent {
         hasFocus: focusStatus,
       }),
       () => {
+        const { hasFocus } = this.state;
         if (focusCallback) {
-          const { hasFocus } = this.state;
           focusCallback(hasFocus);
         }
+
+        clearTimeout(this.focusTimeout);
+
+        this.focusTimeout = setTimeout(() => {
```

Comment:
this use of setTimeout is a little hacky but it's in place because the button should only switch back if both range inputs have been blurred and if a user clicks the other input or tabs between them and then focus is checked then there is a gap where focus is lost before the next focus event fires. By using setTimeout in the setState callback we wait until after the first state operation and then wait a tick which allows focus to move to the next field and return `hasFocus` to `true`